Composer/plugins/azurePublish/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "azurePublish",
+  "name": "plugin-azure-publish",
```

Comment:
I am not sure which is better, it's modified by Ben. But every time we change this name, the old deploy profile would throw error, because it's the key to find the plugin.

Reply:
this got changed when the files were moving back and forth - i think it should be "azurePublish"

Reply:
done